promptsource/templates/dbpedia_14/templates.yaml

```diff
@@ -74,15 +46,16 @@ templates:
     - Written Work
     answer_choices_key: null
     id: 824ecb55-ecad-40c2-8033-f2fa0add2ddf
-    jinja: '{{title}} - {{content}} The text refers to what category? ||| {{ answer_choices[label]
+    jinja: '{{title}} - {{content}} What does the paragraph talk about? ||| {{ answer_choices[label]
```

Comment:
Hmm… Wording it in an interrogative sentence like this still reads weird. Like a normal human is expected to answer the entity's name, not a category name. A declarative sentence like what I suggested earlier is better, but that'd require adding determiner agreement "a company" vs. "an athlete" in the targets. Let's just delete this template for now.

Reply:
What if I add the indefinite article in the answer choices?